packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -542,12 +635,7 @@ export default (async function PgIntrospectionPlugin(
       }
     };
     pgClient.on("notification", listener);
-    introspectionResultsByKind = await introspect();
+    //TODO not sure why we would be doing introspection here (this won't change what's already attached to build)
+    //introspectionResultsByKind = await introspect();
```

Comment:
After you trigger a watch in graphile, all the watch registration functions are called and then a new schema is built. From then onwards the schema will be up to date with the watch fixtures. We must introspect here otherwise we'll miss the following situation:

- [user-code] build initial schema
- [db] edit a table
- [user-code] enable watch mode

^ the "edit a table" results won't be represented.

Note in graphile you can watch/unwatch a schema at any time:  https://www.graphile.org/graphile-build/schema-builder/#watchschemalistener

Reply:
I might be missing something, but I'm fairly sure that that case is covered because we have the ``this.emit("schema", this.buildSchema())`` at the very end of ``watchSchema`` of ``SchemaBuilder``. The already guarantees a schema rebuild upon calling ``watchSchema`` (unless an error is thrown in the try block).